tests/rule_based_profiler/config/test_base.py

```diff
@@ -0,0 +1,307 @@
+import pytest
+from ruamel.yaml.comments import CommentedMap
+
+from great_expectations.marshmallow__shade.exceptions import ValidationError
+from great_expectations.rule_based_profiler.config.base import (
```

Comment:
can we put these types in the config module rather than .base?

Reply:
so in `config/__init__.py`?

Reply:
done!